sheetshuttle/main.py

```diff
@@ -59,5 +68,14 @@ def load_plugin(directory: str, name: str):
     return plugin_source, my_plugin
 
 
+def load_json_file(file_path):
+    """Return the contents of a json file in the file path."""
+    if not file_path:
+        return {}
+    with open(file_path, encoding="uts-8") as read_file:
+        data = json.load(read_file)
+        return data
```

Comment:
We should handle the error where the file does not exist and raise that error with a message. Please add that in a `try` statement